### ISSUE TITLE
Test result comment not shown

### ISSUE THREAD

**Issue by lordpomeroy** [ImisDevelopers/1_011_a_infektionsfall_uebermittellung#395]
**Issue:**
The test result comment that can be added is not shown in the patient details.

**Steps to reproduce:**
Add a test result to a patient and add a comment.
Open the patient's detail page.

**Desired Solution:**
The comment should be saved(?) and shown.

**Comment by jabrandes** [comment]
Indeed, the `comment` field was not saved in the backend. I constructed a fix for that, but I am not sure whether the behaviour I coded is desired. 

There are two types of comments that can be entered for tests. The first is entered by the testing site on probe association, the second by the lab performing the test. The question that arose to me is whether both types of comments need to be stored separately such that the latter is not overriding the former. I would greatly appreciate other opinions on that specific issue.
